test/Microsoft.Extensions.CommandLineUtils.Tests/Microsoft.Extensions.CommandLineUtils.Tests.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
     <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(DeveloperBuild)' == 'true' ">$(DeveloperBuildTestTfms)</TargetFrameworks>
```

Comment:
Why not turn this into :
```
<TargetFrameworks>$(StandardTestTfms)</TargetFrameworks>
```
and do all the conditional logic in `test/Directory.Build.props`?

Reply:
Sure.